fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/Value.java

```diff
@@ -434,91 +442,6 @@ default Optional<Value> promoteToTypeMaybe(@Nonnull final Type type) {
         return Optional.empty();
     }
 
-    @Nonnull
```

Comment:
changes from this line seem to be irrelevant to your PR, I am not sure what changed, but can you check whether you can revert them?

Reply:
This fixes a code style issue, where we normally expect all inner classes to be last in the enclosing class (after methods, etc.). I think I added a static method (which come after the selfish methods, but before the nested classes) or a nested class and cleaned that up.